scripts/generate-icon.py

```
@@ -0,0 +1,46 @@
+import numpy as np
+from PIL import Image
+import os, math
+
+SRC = "/tmp/tetonne1.png"
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/tetonne1.png"

(S108)